aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/MSKFirehoseEvent.java

```diff
@@ -0,0 +1,68 @@
+package com.amazonaws.services.lambda.runtime.events;
+
+import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.Map;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Created by vermshas on 6/28/24.
+ * {
```

Comment:
Please remove this example and link the docs

Reply:
Removed the example, I am not able to find any document stating the specific event.